rdr_service/resource/generators/participant.py

```diff
@@ -928,15 +928,9 @@ def _make_sample_dict_from_row(bss=None, bos=None):
                    bo.processed_site_id, (select google_group from site where site.site_id = bo.processed_site_id) as processed_site,
                    bo.finalized_site_id, (select google_group from site where site.site_id = bo.finalized_site_id) as finalized_site,
                    bo.finalized_time,
-                   case when exists (
-                     select bmko.participant_id
-                     from biobank_mail_kit_order bmko
-                     where bmko.biobank_order_id = bo.biobank_order_id
-                        and bo.participant_id = bmko.participant_id
-                        and bmko.associated_hpo_id is null
-                   )
-                   then 1 else 0 end as dv_order
-             from biobank_order bo where participant_id = :p_id
+                   case when bmko.id is not null then 1 else 2 end as collection_method
```

Comment:
Based on this SQL in `participant_summary_dao.py` are we folding a potential SampleCollectionMethod.UNSET scenario into one of the other method values here?   Could we be introducing deltas in the QC counts for RDR vs. PDR?

https://github.com/all-of-us/raw-data-repository/blob/964f9f2930eb38309ee57453a6036da4a55521c7/rdr_service/dao/participant_summary_dao.py#L198

Reply:
Never mind, I think it's taken care of by the "orderless sample" logic below...

Reply:
As I see it, if there is a biobank order record the only two possible values are 1 or 2.  The exception is when we have orphaned sample records without a biobank order.  In the case of orphans, I defaulted to UNSET.  Based on the SQL in the 'participant_summary_dao.py' file, I can't ever see the UNSET else code path being used as the 'biobank_order' table is the left side of the join.